### PR TITLE
fix(widget-v2): address setting page issue

### PR DESCRIPTION
**Pull request by nikaaru**

### CODE REVIEW

File: widget/embedded/src/components/Slippage/Slippage.styles.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { styled } from '@rango-dev/ui';
+
+export const BaseContainer = styled('div', {
+  padding: '5px 10px',
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.styles.ts:4]: if we have these measures in our theme please use $

File: widget/embedded/src/components/Slippage/Slippage.styles.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  marginBottom: '30px',
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.styles.ts:5]: Don't use margin. Alternatively, you can use padding or <Divider >

File: widget/embedded/src/components/Slippage/Slippage.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  Head,
+  SlippageChipsContainer,
+  SlippageTooltipContainer,
+} from './Slippage.styles';
+
+interface TitleContainerProps {
+  title: string;
+}
+
+function TitleContainer(props: TitleContainerProps) {
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.tsx:30]: Why do we need a component for a simple Typography?

File: widget/embedded/src/components/Slippage/Slippage.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+function SlippageTooltipContent() {
+  return (
+    <SlippageTooltipContainer>
+      <Typography variant="label" size="small" color="neutral900">
+        {i18n.t(
+          'Your transaction will be reverted if the price changes unfavorably by more than this percentage'
+        )}
+        <br />
+        <br />
+        <b>{i18n.t('Warning')}</b>:{' '}
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.tsx:48]: Remove {' '}

File: widget/embedded/src/components/Slippage/Slippage.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+function TitleContainer(props: TitleContainerProps) {
+  const { title } = props;
+  return (
+    <Typography variant="title" size="xmedium" color="neutral900">
+      {title}
+    </Typography>
+  );
+}
+
+function SlippageTooltipContent() {
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.tsx:39]: I think we don't need such component separately, but if you insist you can put it in another file

File: widget/ui/src/components/Chip/Chip.styles.ts [hunk 6]
```diff
@@ -9,12 +9,16 @@ export const ChipContainer = styled('button', {
...
   backgroundColor: '$neutral200',
-  border: 'none',
+  border: '1px solid transparent',
 
   '&:hover': {
     backgroundColor: '$surface600',
   },
 
+  '&:focus': {
+    outline: '1px solid gray',
```
Comment (samobasquiat) [inline widget/ui/src/components/Chip/Chip.styles.ts:19]: do we have gray in our theme?

File: widget/ui/src/components/Tooltip/Tooltip.types.ts [hunk 7]
```diff
@@ -1,5 +1,7 @@
+import type { ReactNode } from 'react';
+
 export interface PropTypes {
-  content: string;
+  content: string | ReactNode;
```
Comment (samobasquiat) [inline widget/ui/src/components/Tooltip/Tooltip.types.ts:4]: ReactNode includes string as well.

File: widget/embedded/src/components/Slippage/Slippage.tsx (hunk 3 above)
Comment (nikaaru) [reply widget/embedded/src/components/Slippage/Slippage.tsx:30]: It was used several times, after the file was separated, it was missed.
Done

File: widget/embedded/src/components/Slippage/Slippage.tsx (hunk 5 above)
Comment (nikaaru) [reply widget/embedded/src/components/Slippage/Slippage.tsx:39]: Done

File: widget/ui/src/components/Chip/Chip.styles.ts (hunk 6 above)
Comment (nikaaru) [reply widget/ui/src/components/Chip/Chip.styles.ts:19]: I didn't find anything in figma, I put this temporarily

File: widget/ui/src/components/Chip/Chip.styles.ts [hunk 8]
```diff
@@ -19,6 +19,10 @@ export const ChipContainer = styled('button', {
     backgroundColor: '$$color',
   },
 
+  '&:focus': {
```
Comment (samobasquiat) [inline widget/ui/src/components/Chip/Chip.styles.ts:22]: I think you can remove the outline in the selected state. in the current version selected chips have a border & outline simultaneously.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx [hunk 9]
```diff
@@ -145,14 +150,28 @@ export function LiquiditySourcePage({
...
 
-        <ListContainer>
-          <List
-            type={
-              <ListItemButton title="_" id="_" onClick={() => console.log()} />
-            }
-            items={filteredList}
-          />
-        </ListContainer>
+        {!filteredList.length && !!searchedFor ? (
```
Comment (samobasquiat) [inline widget/embedded/src/pages/LiquiditySourcePage.tsx:153]: if you sync with the latest version you can use the Search component and have the close icon to remove the text.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx [hunk 10]
```diff
@@ -145,14 +150,28 @@ export function LiquiditySourcePage({
...
+          <>
+            <Divider size={32} />
+            <NotFound
```
Comment (samobasquiat) [inline widget/embedded/src/pages/LiquiditySourcePage.tsx:156]: the place of NotFound is not matched with figma
image

File: widget/embedded/src/components/Slippage/Slippage.tsx [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+        <Typography variant="title" size="xmedium">
+          {i18n.t('Slippage tolerance per swap')}
+        </Typography>
+        <Divider direction="horizontal" size={4} />
+        <Tooltip side="top" content={<SlippageTooltipContent />}>
+          <InfoIcon color="gray" />
+        </Tooltip>
+      </Head>
+      <SlippageChipsContainer>
+        {SLIPPAGES.map((slippageItem, index) => {
```
Comment (samobasquiat) [inline widget/embedded/src/components/Slippage/Slippage.tsx:62]: the width of slippages isn't matched with Figma. custom slippage as well.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx [hunk 12]
```diff
@@ -145,14 +150,28 @@ export function LiquiditySourcePage({
...
+              title={i18n.t('No results found')}
+              description={i18n.t('Try using different keywords')}
+            />
+          </>
+        ) : (
+          <ListContainer>
+            <List>
```
Comment (samobasquiat) [inline widget/embedded/src/pages/LiquiditySourcePage.tsx:163]: I couldn't find a separator line. In Figma it has color.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx [hunk 13]
```diff
@@ -99,7 +100,11 @@ export function LiquiditySourcePage({
...
       end: <Checkbox checked={selected} />,
-      title: <TitleContainer title={i18n.t(groupTitle)} />,
+      title: (
+        <Typography variant="title" size="xmedium">
+          {i18n.t(groupTitle)}
+        </Typography>
+      ),
       ...sourceItem,
     };
   });
```
Comment (samobasquiat) [inline widget/embedded/src/pages/LiquiditySourcePage.tsx:110]: select All & Deselect All buttons have a hover state in Figma

**Review by samobasquiat — COMMENTED** [review]
After clicking on checkboxes in liquidities, the place of the item changes a little.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx (hunk 9 above)
Comment (nikaaru) [reply widget/embedded/src/pages/LiquiditySourcePage.tsx:153]: I pulled the last version of alpha, and now is synced.
thanks for your update.

File: widget/embedded/src/pages/LiquiditySourcePage.tsx (hunk 10 above)
Comment (nikaaru) [reply widget/embedded/src/pages/LiquiditySourcePage.tsx:130]: Done

**Review by samobasquiat — APPROVED** [review]

**Pull request merged**